src/pudl_archiver/metadata/sources.py

```diff
@@ -33,7 +106,35 @@
         ),
         "working_partitions": {
             "years": [1991, 1994, 1998, 2002, 2006, 2010, 2014, 2018]
-        },  # Census DP1 is monolithic.
+        },
+        "keywords": sorted(
+            {
+                "manufacturing",
```

Comment:
For consistency with the other `EIA * -- ` prefix datasets
```suggestion
                "eia",
                "manufacturing",
```